src/Humanizer/Localisation/TimeToClockNotation/DeTimeOnlyToClockNotationConverter.cs

```diff
@@ -0,0 +1,93 @@
+﻿#if NET6_0_OR_GREATER
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Humanizer.Localisation.TimeToClockNotation;
+
+public class DeTimeOnlyToClockNotationConverter : ITimeOnlyToClockNotationConverter
```

Comment:
```suggestion
internal class DeTimeOnlyToClockNotationConverter : ITimeOnlyToClockNotationConverter
```
Class should probably be internal like ie. `FrTimeOnlyToClockNotationConverter`, so it doesn't have to be added to the approved public API.

Reply:
OK I'll do it.